src/main/java/kmql/Database.java

```diff
@@ -137,6 +150,37 @@ public boolean tableInitialized(String name) {
         return getTable(name).initialized;
     }
 
+    /**
+     * Return tables contained in this database
+     * @return the set of {@link Table}
+     */
+    public Set<Table> tables() {
```

Comment:
I think `Table` instance should be managed completely internally by Database to avoid any unintentional call against its methods. So this should return only names as `Set<String>` rather than `Table`s?